tests/test_modeling_common.py

```diff
@@ -2559,7 +2561,7 @@ def test_cpu_offload(self):
 
             model_size = compute_module_sizes(model)[""]
             # We test several splits of sizes to make sure it works.
-            max_gpu_sizes = [int(p * model_size) for p in self.model_split_percents]
+            max_gpu_sizes = [int(p * model_size) for p in self.model_split_percents[1:]]
```

Comment:
we need at least `0.6` now after changing to `layers=2`, but I don't want  to add this new item to `self.model_split_percents`.

Reply:
It's 0.7 now with your changes.